content/en/platform/corda/5.0/application-networks/managing/member-suspension/_index.md

```diff
@@ -0,0 +1,94 @@
+
+## Suspend a Member
+
+You can use the <a href="../../reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__suspend">POST method of the `/api/v1/members/{holdingidentityshorthash}/suspend` endpoint</a> to suspend a member of a group:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```shell
+curl -u $REST_API_USER:$REST_API_PASSWORD -X 'POST' "$API_URL/mgm/$MGM_HOLDING_ID/suspend" -H 'Content-Type: application/json' \
+ -d '{"x500Name": '\"$MEMBER_X500_NAME\"', "serialNumber": "<serial-number>"}'
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+Invoke-RestMethod -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)}`
+  -Method Post -Uri $API_URL/mgm/$MGM_HOLDING_ID/suspend -Body (ConvertTo-Json -Depth 1 @{
+  x500Name = $MEMBER_X500_NAME; serialNumber = <serial-number>})
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+`<serial-number>` is the current serial number of the member, as retrived by the [Member Lookup endpoint]({{< relref "#query-for-members">}}). The `<serial-number>` is optional. If it is not specified, the latest serial number is used. However, we recommend always specifying the serial number in the request to avoid suspending a member based on outdated information. If the serial number does not match, the method returns a `409 CONFLICT`. This can happen if another process has updated the member information, before the suspension operation. If this occurs, you can query the Member Lookup endpoint again and decide whether to proceed with the operation.
+
+## Activate a Member
+
+You can use the <a href="../../reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__activate">POST method of the `/api/v1/members/{holdingidentityshorthash}/activate` endpoint</a> to re-activate a suspended member of a group:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```shell
+curl -u $REST_API_USER:$REST_API_PASSWORD -X 'POST' "$API_URL/mgm/$MGM_HOLDING_ID/activate" -H 'Content-Type: application/json' \
+ -d '{"x500Name": '\"$MEMBER_X500_NAME\"', "serialNumber": "<serial-number>"}'
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)}`
+  -Method Post -Uri $API_URL/mgm/$MGM_HOLDING_ID/activate -Body (ConvertTo-Json -Depth 1 @{
+  x500Name = $MEMBER_X500_NAME; serialNumber = <serial-number>})
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+`<serial-number>` is the current serial number of the member, as retrived by the [Member REST endpoint]({{< relref "#query-for-members">}}). The `<serial-number>` is optional. If it is not specified, the latest serial number is used. However, we recommend always specifying the serial number in the request to avoid suspending a member based on outdated information. If the serial number does not match, the method returns a `409 CONFLICT`.
```

Comment:
```suggestion
`<serial-number>` is the current serial number of the member, as retrieved by the [Member REST endpoint]({{< relref "#query-for-members">}}). The `<serial-number>` is optional. If it is not specified, the latest serial number is used. However, we recommend always specifying the serial number in the request to avoid suspending a member based on outdated information. If the serial number does not match, the method returns a `409 CONFLICT`.
```